app/src/main/java/com/mapbox/maps/testapp/examples/terrain3D/SantaCatalinaActivity.kt

```diff
@@ -132,14 +132,15 @@ class SantaCatalinaActivity : AppCompatActivity() {
         response: Response<DirectionsResponse>
       ) {
         response.body()?.let { body ->
-          body.routes()[0].geometry()?.let {
-            animateRoute(
-              LineString.fromPolyline(it, PRECISION_6)
-            )
-            return
+          if (body.routes().isNotEmpty()) {
```

Comment:
Or just `body.routes().firstOrNull()?.geometry()?.let {`

Reply:
I've spent too much time already waiting for CI, let's leave as is, we can revisit it later during some kind of example cleanup.